dashboard/app/views/levels/_admin.html.haml

```diff
@@ -25,10 +25,15 @@
                   %li= link_to 'pre-draw', edit_blocks_level_path(@level, :predraw_blocks)
               - unless @level.uses_droplet?
                 %button{type: 'button', onclick: 'window.levelbuilder.copyWorkspaceToClipboard()', class: 'btn btn-default btn-sm'}
-                  Copy workspace to clipboard
+                  Copy workspace XML to clipboard
                 %br
                 %button{type: 'button', onclick: 'window.levelbuilder.copySelectedBlockToClipboard()', class: 'btn btn-default btn-sm'}
-                  Copy selected block to clipboard
+                  Copy selected block XML to clipboard
+                %br
+                %button{type: 'button', onclick: 'window.levelbuilder.pasteBlocksToWorkspace()', class: 'btn btn-default btn-sm'}
+                  Paste XML block(s)* to workspace
```

Comment:
Am I understanding correctly that to use this button, you need to have copied using one of the copy buttons in the same menu (rather than context menu or keyboard shortcut)? Could we make that clear somehow?

Reply:
Yes, that is correct, because they use separate storage entries and a different data format. The buttons that work together are grouped together and styles the same way. Do you have any suggestions for other things we could do to make the relationship clearer?

Reply:
Ah, yeah I think that's what I was going for, but it's probably not clear/needed. It is a very small space, so I don't think we want to go too overboard. I think I'll probably just pull out the asterisk.